content/java/concepts/math-methods/terms/toDegrees/toDegrees.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: '.toDegrees()'
+Description: 'Returns the measurement in degrees of an angle in radians.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.toDegrees()`** method is a built-in math function in Java which is used to convert an angle measured in radians to an approximately equivalent angle measured in degrees.
+
+## Syntax
+
+```pseudo
+Math.toDegrees(double x)
+```
+
+The return type of the `.toDegrees()` method is a double. The data type of the `x` parameter is also a double.
```

Comment:
"double" is a keyword in Java. It should be surrounded with backticks like so: `double`.